website/docs/Pipelines/Exome_Germline_Single_Sample_Pipeline/exome.methods.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 2
+---
+
+# Exome Germline Single Sample v2.4.4 Methods
+
+The following contains a detailed methods description outlining the pipeline’s process, software, and tools that can be modified for a publication methods section.
+
+## Detailed Methods
+
+Preprocessing and variant calling was performed using the ExomeGermlineSingleSample 2.4.4 pipeline using Picard 2.23.8, GATK 4.1.8, and Samtools 1.11 with default tool parameters unless otherwise specified. All reference files are available in the public [Broad References Google Bucket](https://console.cloud.google.com/storage/browser/gcp-public-data--broad-references/hg38/v0). The pipeline follows GATK Best Practices as previously described ([Van der Auwera & O'Connor, 2020](https://www.oreilly.com/library/view/genomics-in-the/9781491975183/)) as well as the Functional Equivalence specification ([Regier et al., 2018](https://www.nature.com/articles/s41467-018-06159-4)).
```

Comment:
I like that the version is in here, but we just need to make sure that it gets updated.

Reply:
@kbergin is there a way to automate the version here based on the changelog?  Or maybe we add it to some notes somewhere?

Reply:
Maybe that's an issue for @kbergin

Reply:
I will add a ticket to explore it. Definitely a general problem, keeping documentation up to date.

Reply:
https://broadinstitute.atlassian.net/browse/GL-1642